posts/2022-11-22-22.0.0.12.adoc

```diff
@@ -0,0 +1,267 @@
+	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.invokePartitionsRevoked(ConsumerCoordinator.java:315)
+----
++
+This issue has been resolved and the partition assignment succeeds correctly and without the above errors.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/21808[Provide a way for Custom User Registries to use the uniqueId instead of the securityName]
++
+When using a `CustomUserRegistry` and making a programmatic API call like `getRemoteUser`, `getCallerPrincipal` or `getUserPrincipal`, Open Liberty was incorrectly returning the `securityName`.
++
+The issue now resolved and the methods `getRemoteUser`, `getCallerPrincipal` or `getUserPrincipal` now correctly return the `displayName`.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/23252[AmbiguousResolutionException when same class is present twice and certain features are used]
++
+When a class is present twice in an application and is used as a CDI managed bean and certain features are used, CDI may sometimes report an `AmbiguousResolutionException`.
++
+[source]
+----
+Caused by: org.jboss.weld.exceptions.AmbiguousResolutionException: WELD-001318: Cannot resolve an ambiguous dependency between: 
+  - Managed Bean [class com.ibm.wssvt.acme.annuity.common.business.jaxrs.MyJAXBResolver] with qualifiers [@Any @Default],
+  - Managed Bean [class com.ibm.wssvt.acme.annuity.common.business.jaxrs.MyJAXBResolver] with qualifiers [@Any @Default]
+----
++
+This issue is now resolved.  The same bean is no longer included twice and, therefore, the exception is no longer thrown.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/23225[IllegalStateException in DynaCache when app server is stopping]
++
+Sometimes while the Liberty server is shutting down with the `webCache-1.0` feature enabled, an FFDC is created by a DynaCache daemon thread:
++
+[source]
+----
+Stack Dump = java.lang.IllegalStateException: ScheduledExecutorService service is unavailable
+ at com.ibm.ws.cache.Scheduler.getScheduledExecutorService(Scheduler.java:93)
+ at com.ibm.ws.cache.Scheduler.createNonDeferrable(Scheduler.java:108)
+ at com.ibm.ws.cache.RealTimeDaemon.alarm(RealTimeDaemon.java:107)
+ at com.ibm.ws.cache.RealTimeDaemon$2.run(RealTimeDaemon.java:111)
+ at com.ibm.ws.cache.Scheduler$WrappedRunnable.run(Scheduler.java:130)
+ ...
+----
++
+The issue is now resolved.  DynaCache no longer attempts to create a schedule when the Liberty server is stopping and the FFDC is not generated.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/23186[IdentityStore validate method not getting called for BasicAuthentication request]
++
+Some Basic Authentication requests fail due to `IdentityStore` not being called correctly.  This a result of Liberty's `BasicHttpAuthenticationMechanism` using `BasicAuthenticationCredential` instead of `UsernamePasswordCredential`. That credential object is passed down to the `IdentityStore` validate method call as a parameter.
```

Comment:
```suggestion
Some Basic Authentication requests fail due to `IdentityStore` not being called correctly.  This a result of Liberty's `BasicHttpAuthenticationMechanism` using `BasicAuthenticationCredential` instead of `UsernamePasswordCredential`. That credential object is passed down to the `IdentityStore` `validate` method call as a parameter.
```